Signal/test/util/CDSSigningCertificateTest.m

```diff
@@ -17,25 +17,32 @@ @implementation CDSSigningCertificateTest
 - (void)testParsing_good
 {
     NSString *pem = [self certificatesPem_good];
-    CDSSigningCertificate *_Nullable certificate = [CDSSigningCertificate parseCertificateFromPem:pem];
+    NSError *error;
+    CDSSigningCertificate *_Nullable certificate = [CDSSigningCertificate parseCertificateFromPem:pem error:&error];
     XCTAssertNotNil(certificate);
+    XCTAssertNil(error);
 }
 
 - (void)testParsing_bad
 {
     NSString *pem = [self certificatesPem_bad];
-
-    XCTAssertThrows([CDSSigningCertificate parseCertificateFromPem:pem]);
+    NSError *error;
+    CDSSigningCertificate *_Nullable certificate = [CDSSigningCertificate parseCertificateFromPem:pem error:&error];
```

Comment:
firstly, this method doesn't throw anymore, but secondly, but checking specific error types we can have a better sense about which paths are tested.